config/index.md

```diff
@@ -431,6 +431,39 @@ export default async ({ command, mode }) => {
 
   File system watcher options to pass on to [chokidar](https://github.com/paulmillr/chokidar#api).
 
+### server.fsServe.strict
+
+- **Experimental**
+- **Type:** `boolean`
+- **Default:** `false` (will change to `true` in future versions)
+
+  Restrict serving files outside of workspace root.
+
+### server.fsServe.root
+
+- **Experimental**
+- **Type:** `string`
+
+  Restrict files that could be served via `/@fs/`. When `server.fsServe.strict` is set to `true`, accessing files outside this directory will result in a 403.
+
+  Vite will search for the root of the potential workspace and use it as default. A valid workspace met the following conditions, otherwise will fallback to the [project root](/guide/#index-html-and-project-root).
```

Comment:
```suggestion
  Vite will search for the root of the potential workspace and use it as default. A valid workspace met the following conditions, otherwise will fallback to the [project root](/guide/#index-html-and-project-root).

```

https://github.com/vitejs/vite/commit/d36e10e#diff-f7d4a01431aa2057a8d0c4ff1b1c233af89820b533c5fe3a64bc813eee69bab0R449

Vite will search for... の次に空行が追加されてる箇所、これが抜けてるっぽいです。

Reply:
見落としていました。ありがとうございます:pray: